src/api/config/routes/webui_routes.rb

```diff
@@ -279,6 +277,11 @@
         resource :files, controller: 'webui/packages/files', only: [:new, :create, :update], constraints: cons
         put 'toggle_watched_item', controller: 'webui/watched_items', constraints: cons
         resource :badge, controller: 'webui/packages/badge', only: [:show], constraints: cons.merge(format: :svg)
+        resources :repositories, only: [], param: :name do
+          resources :binaries, controller: 'webui/packages/binaries', only: [:index], constraints: cons
+          # Binaries with the exact same name can exist in multiple architectures, so we have to use arch param here additionally
+          resources :binaries, controller: 'webui/packages/binaries', only: [:show], constraints: cons, param: :filename, path: 'binaries/:arch/'
```

Comment:
💭  I'm not sure I like this, feels hack'ish to abuse `resources` like that to draw the route....

Reply:
hm, I fear this will break a lot of links...

Reply:
I guess I'm just grumpy 😉

Reply:
I meant external ones

Reply:
okay then :)

Reply:
I guess leave it as it was then...